src/CollAction/Models/Project.cs

```diff
@@ -60,7 +60,11 @@ public class Project
         [Required]
         [MaxLength(2048)]
         public string CreatorComments { get; set; }
-        
+
+        public int? ImageFileId { get; set; }
```

Comment:
BannerImage(File)Id?

I'd imagine it's possible for more images to be associated with the project table.